docs/user/environment-variables.md

```diff
@@ -43,8 +43,8 @@ The installer accepts a number of environment variable that allow the interactiv
 ## Platform-Specific
 
 * `AWS_PROFILE`:
-     The AWS profile that corresponds to value in `${HOME}/.aws/credentials`.  If not provided, the default is "default".
-* `OPENSHIFT_INSTALL_AWS_REGION`:
+    The AWS profile that corresponds to value in `${HOME}/.aws/credentials`.  If not provided, the default is "default".
+* `AWS_DEFAULT_REGION`:
```

Comment:
wouldn't `AWS_PROFILE` be enough?

Reply:
And if we keep `AWS_DEFAULT_REGION` people who have setup this env will see no option to choose a separate  region. and will have to unset this default aws env. Which IMO is not great UX.

Using this to choose the default region in select is :+1:

Reply:
> wouldn't `AWS_PROFILE` be enough?

That's enough to look up `~/.aws/config` entries.  But it's not the only place the `aws` CLI tool, Terraform, etc. look for region configuration.  I'd rather follow them.

> And if we keep AWS_DEFAULT_REGION people who have setup this env will see no option to choose a separate region. and will have to unset this default aws env. Which IMO is not great UX.

If I run `aws ec2 describe-instances`, etc., it doesn't prompt me for "what region should I look at?" ;).  And it doesn't seem very likely that they'd want the bulk of their AWS activity in one region but their OpenShift cluster off in another.  How many folks do lots of region-hopping *and* set `AWS_DEFAULT_REGION`?  If I was doing lots of region hopping, I'd rely on `~/.aws/config` and `AWS_PROFILE`.

But if you don't buy that argument and would rather me keep `OPENSHIFT_INSTALL_AWS_REGION` around as the only way to bypass the wizard, with `AWS_DEFAULT_REGION` and all the standard locations only feeding the wizard default, I can do that.

Reply:
> Yeah, IMO that's more appropriate use of those envs.

Ok, done with ca6f931 -> ed50028.